.travis.yml

```diff
@@ -3,6 +3,12 @@ language: python
 python:
   - "2.7"
   - "3.4"
+cache:
+  pip: true
+  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # along with the indexed db of intervals and ID mappings and pickles
+  # of sequence dictionaries
+  directories: /home/travis/.cache/pyensembl/
```

Comment:
is it also worth caching the FM indices? that'll live at `/home/travis/.cache/vaxrank`, though I can rename the directory to something more descriptive